core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.component.ts

```diff
@@ -169,7 +169,9 @@ export class CustomizeInteractionModalComponent
   }
 
   getTitle(interactionId: string): string {
-    return INTERACTION_SPECS[interactionId].name;
+    if (INTERACTION_SPECS[interactionId] !== undefined) {
```

Comment:
In which scenario would the value of `INTERACTION_SPECS[interactionId]` be undefined?

Reply:
What frontend test? What case is failing exactly?

Reply:
for the front-end test thing, now I have removed the check `if (INTERACTION_SPECS[interactionId] !== undefined)` because now no test is failing on removing this. I faced some issue sometime before but now everything seems fine on removing this check!